File: android/src/main/kotlin/com/tahamalas/custom_image_picker/PhoneAlbum.kt
package com.tahamalas.custom_image_picker

class PhoneAlbum(val id: String, val name: String, val coverUri: String, val photosCount: Int) {

    override fun toString(): String {
        return "{\"id\": \"$id\", \"name\": \"$name\", \"coverUri\": \"$coverUri\", \"photosCount\": $photosCount} "
    }
}

File: ios/Classes/PhoneAlbum.swift
//
//  PhoneAlbum.swift
//  custom_image_picker
//
//  Created by Malas Taha on 12/29/19.
//

import Foundation

class PhoneAlbum {
    
    let id:String
    let name:String
    let coverUri:String
    let photosCount: Int
    
    init(id: String, name: String, coverUri: String, photosCount: Int) {
        self.id = id
        self.name = name
        self.coverUri = coverUri
        self.photosCount = photosCount
    }
    
    func toJson() -> String {
        return "{\"id\": \"\(id)\", \"name\": \"\(name)\", \"coverUri\": \"\(coverUri)\", \"photosCount\": \(photosCount)}"
    }
    
}
